tests/2018-01-01/Microsoft.Network.Microsoft.EventGrid.tests.json

```diff
@@ -0,0 +1,92 @@
+{
+  "tests": [
```

Comment:
>tests [](start = 3, length = 5)

would be good if can add a positive test case.

Reply:
added a positive test case